yt/yt/server/node/exec_node/chunk_cache.cpp

```diff
@@ -327,7 +327,7 @@ class TChunkCache::TImpl
 
             futures.push_back(
                 BIND(&TImpl::InitializeLocation, MakeStrong(this))
-                    .AsyncVia(location->GetAuxPoolInvoker())
+                    .AsyncVia(location->GetAuxPoolInvoker({}, {})) // TODO what to pass here
```

Comment:
Perhaps it is worth using the default parameters here (in the method)?

Reply:
I made them explicit, to make sure that all code occurences are adjusted("colored")
Yes we can disable explicit coloring and default to WorkloadCategory Idle and empty SessionId

Reply:
In the next version I removed changes to GetAuxPoolInvoker, they actually had no effect. (passed parameters were ignored)